src/collectors/github_runners.rs

```diff
@@ -0,0 +1,180 @@
+use crate::{http, http::is_token_flagged, Config};
+use anyhow::Error;
+use log::{debug, error, warn};
+use prometheus::core::Desc;
+use prometheus::proto::MetricFamily;
+use prometheus::{core::Collector, IntGauge, Opts};
+use std::collections::HashMap;
+use std::sync::{Arc, RwLock};
+use tokio::time::Duration;
+
+const GH_RUNNERS_ENDPOINT: &str = "https://api.github.com/repos/{owner_repo}/actions/runners";
+
+#[derive(Debug, serde::Deserialize)]
+struct ApiResponse {
+    total_count: usize,
+    runners: Vec<Runner>,
+}
+
+#[derive(Debug, serde::Deserialize)]
+struct Runner {
```

Comment:
We don't need the struct to map all the fields provided by the GitHub API: if there is a field in the response that's not present in the struct serde will just skip it.

Reply:
removed field not used from the struct in the `rebase` commit. 
if we see if there's anything that needs to get back lemme know.